src/ethereum/truffle/rad2sol.js

```diff
@@ -31,17 +33,35 @@ const steps = [
   pack,
   (request) => intoProtoBuf(request, schema),
   buff => buff.toString("hex"),
+  (hex, i) => {
+    const priceName = requestNames[i].replace(/\.js/, "")
+    migrationsRadons[`${priceName}Feed`] = {
+      ERC2362ID: 
+        `Price-${priceName.replace("Price", "")
+          .split(/(?=[A-Z])/)
+          .map(s => s.toUpperCase())
+          .join("/")
+        }-3`,
```

Comment:
We shouldn't assume that contracts produced by `rad2sol` will be `ERC2362` price feeds. And even if that was the case, we can't assume either that the number of decimal digits will be `3`.